app/assets/stylesheets/application/organisms/_item_list.scss

```diff
@@ -0,0 +1,111 @@
+.item-list {
+  background-color: $pale-gray-color;
+  width: 320px;
+
+  .label-container {
+    overflow: visible;
+    position: relative;
+    text-align: center;
+    z-index: 3;
+  }
+
+  label {
+    width: 120px;
+  }
+
+  .label-content-1 {
+    margin-right: 14px;
+  }
+
+  .content-container {
+    height: 380px;
+    position: relative;
+  }
+
+  .toggle {
+    left: -9999px;
+    position: absolute;
+  }
+
+  .toggle-content-1:not(:checked) ~ .content-container > .togglable-content-1 {
+    z-index: 1;
+  }
+
+  .toggle-content-1:checked ~ .content-container > .togglable-content-1 {
+    z-index: 2;
+  }
+
+  .toggle-content-1:checked ~ .label-container > .label-content-1 {
+    @extend .btn-header-active;
+  }
+
+  .toggle-content-2:not(:checked) ~ .content-container > .togglable-content-2 {
+    z-index: 1;
+  }
+
+  .toggle-content-2:checked ~ .content-container > .togglable-content-2 {
+    z-index: 2;
+  }
+
+  .toggle-content-2:checked ~ .label-container > .label-content-2 {
+    @extend .btn-header-active;
+  }
+
+  .togglable-content {
+    background: linear-gradient($pale-gray-color, $white-color, $pale-gray-color);
+    background-color: $pale-gray-color;
+    bottom: 0;
+    left: 0;
+    padding-top: 20px;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+
+  @include breakpoint($desktop) {
+    width: auto;
+
+    .toggle:checked + .togglable-content,
+    .toggle:not(:checked) + .togglable-content {
+      z-index: 1;
+    }
+
+    .content-container {
+      margin: 0 auto;
+      width: 728px;
+    }
+
+    .togglable-content {
+      background: $pale-gray-color;
+      float: left;
+      margin: 0 7px;
+      position: relative;
+      top: 0;
+      width: 350px;
+    }
+
+    .label-content-1 {
+      margin-right: 240px;
+    }
+
+    label {
+      background-color: $primary-color;
+      color: $white-color;
+      cursor: default;
+      position: relative;
+
+      &:after {
+        border-color: $primary-color transparent transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.